scripts/ignore_new_urls.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Used by html-proffer to ignore new/copied/renamed files
+# https://github.com/gjtorikian/html-proofer#ignoring-new-files
+
+main() {
+  MERGEBASE=$(git merge-base origin/staging HEAD | sed 's/ //g')
+
+  SUB='content/en/docs'
+  EXCLUDE_URLS=""
+
+  while IFS= read -r line; do
+  if [[ "$line" =~ .*"$SUB".* ]]; then
+
+              SUBSTR=$(echo $line | sed 's,content/en/docs/,,g')
+              PATTERN=$(echo "${SUBSTR%.*}")
+              URL=$(echo ",/.*${SUBSTR%.*}.*/")
+              URL=$(echo "$URL" | awk '{print tolower($0)}')
+              EXCLUDE_URLS="$EXCLUDE_URLS$URL"
+      fi;
```

Comment:
trivial - but for better readability, if statement should be formatted -- at least line up `fi` with `if` on the same column :) 
```suggestion
  if [[ "$line" =~ .*"$SUB".* ]]; then
    SUBSTR=$(echo $line | sed 's,content/en/docs/,,g')
    PATTERN=$(echo "${SUBSTR%.*}")
    URL=$(echo ",/.*${SUBSTR%.*}.*/")
    URL=$(echo "$URL" | awk '{print tolower($0)}')
    EXCLUDE_URLS="$EXCLUDE_URLS$URL"
  fi;
```

Reply:
Unless there is a reason for doing these separately, I would also consider combining the two statements setting URL - you can just use pipe to combine - you are using pipe anyway on the second line - so it seems you can make this simpler by just combining the two lines:

```
URL=$(echo ",/.*${SUBSTR%.*}.*/" | awk '{print tolower($0)}')
```

Reply:
Yes, looks simpler. I will add the change.